Typos 3.3 (#46)
mulesoft/docs-connector-devkit: modules/ROOT/pages/authorizing-your-connector-with-oauth-2.0.adoc

|authorizationParameters |comma-delimited list of annotations of the form @OAuthAuthorizationParameter(name = "xxx", type = xxx, description = "xxx"). The type cannot be a collection or a complex type. |OPTIONAL |
|verifierRegex |a Java regular expression used to extract the verifier from the service provider response after the resource owner authorizes the consumer |OPTIONAL |code=([^&]+) +
|accessTokenRegex |a Java regular expression used to extract the access token from the service provider response + |OPTIONAL |"access_token":"([^&]+?)" +
|expirationRegex + |a Java regular expression used to extract the expiration time of the access token (in seconds) from the service provider response. If this regular expression is not found in the service provider response (because the regular expression is wrong or the access token never expires), the access token will be treated as if it would never expire. |OPTIONAL |"expires_in":([^&]+?),
|refreshTokenRegex + |a Java regular expression used to extract a refresh token from the callback during the authorization flow. |OPTIONAL |"\"refresh_token\":\"([^&]+?)\""

> |callbackPath |where the service provider only accepts a known redirect URL, assign this parameter to the path inside your domain (denoted by the `fullDomain` environment variable) that will be registered with service provider as a redirect URL. If left empty (meaning the service provider accepts any URL as redirect URL), a random path will be used. |OPTIONAL |_ random path _

Reviewer: ```suggestion
|callbackPath |If the service provider accepts only a known redirect URL for this parameter, use the path inside your domain (denoted by the `fullDomain` environment variable). If left empty and the service provider accepts any URL as a redirect URL for this parameter, a random path is used. |OPTIONAL |_ random path _
```
